common/moduleconfig/testdata/projects/sample/ftl.toml

```diff
@@ -0,0 +1,3 @@
+module = "sample"
+language = "go"
+min-ftl-version = "0.129.2"
```

Comment:
Maybe a question for the group, but do we want the `ftl.toml` files to know about ftl versions? Or just the `ftl-project` files?

Reply:
I think we decided in the sync to go with just `ftl-project.toml` for now, so this can probably be closed out.

Reply:
done!